README.md

```diff
@@ -201,7 +201,7 @@ config :plug, :mimes, %{
 ```
 
 
-And then re-compile plug: (per: http://hexdocs.pm/plug/Plug.MIME.html)
+And then re-compile plug: (per: https://hexdocs.pm/plug/1.1.3/Plug.MIME.html)
```

Comment:
Actually it looks like you do need to recompile mime per the docs @beerlington linked.

@shhavel do you think you could update both the plug > 1.2 and plug < 1.2 instructions?